automl/cloud-client/get_dataset.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def get_dataset(project_id, dataset_id):
+    """Get a dataset."""
+    # [START automl_language_entity_extraction_get_dataset]
+    # [START automl_language_sentiment_analysis_get_dataset]
+    # [START automl_language_text_classification_get_dataset]
+    # [START automl_translate_get_dataset]
+    # [START automl_vision_classification_get_dataset]
+    # [START automl_vision_object_detection_get_dataset]
+    from google.cloud import automl
+
+    # TODO(developer): Uncomment and set the following variables
+    # project_id = 'YOUR_PROJECT_ID'
+    # dataset_id = 'YOUR_DATASET_ID'
+
+    client = automl.AutoMlClient()
+    # Get the full path of the dataset
+    dataset_full_id = client.dataset_path(
+        project_id, "us-central1", dataset_id
+    )
+    dataset = client.get_dataset(dataset_full_id)
+
+    # Display the dataset information
+    print(u"Dataset name: {}".format(dataset.name))
+    print(u"Dataset id: {}".format(dataset.name.split("/")[-1]))
+    print(u"Dataset display name: {}".format(dataset.display_name))
+    print("Dataset create time:")
+    print(u"\tseconds: {}".format(dataset.create_time.seconds))
+    print(u"\tnanos: {}".format(dataset.create_time.nanos))
+    # [END automl_language_sentiment_analysis_get_dataset]
+    # [END automl_language_text_classification_get_dataset]
+    # [END automl_translate_get_dataset]
+    # [END automl_vision_classification_get_dataset]
+    # [END automl_vision_object_detection_get_dataset]
+    print(
+        "Text extraction dataset metadata: {}".format(
+            dataset.text_extraction_dataset_metadata
+        )
+    )
+    # [END automl_language_entity_extraction_get_dataset]
+
+    # [START automl_language_sentiment_analysis_get_dataset]
+    print(
+        "Text sentiment dataset metadata: {}".format(
+            dataset.text_sentiment_dataset_metadata
+        )
+    )
+    # [END automl_language_sentiment_analysis_get_dataset]
+
+    # [START automl_language_text_classification_get_dataset]
+    print(
+        "Text classification dataset metadata: {}".format(
+            dataset.text_classification_dataset_metadata
+        )
+    )
+    # [END automl_language_text_classification_get_dataset]
+
+    # [START automl_translate_get_dataset]
+    print("Translation dataset metadata:")
+    print(
+        u"\tsource_language_code: {}".format(
```

Comment:
I think we can probably drop python2 support at this point, but if not please use `from future import unicode_literals` instead.

Reply:
Can we switch to inline?
```
print(f"\tsource_language_code: {dataset.translation_dataset_metadata.source_language_code}")
```

Reply:
errr actually if Kokoro is running 2.7 still, can't do the `f` prefix

Reply:
fstrings is different than unicode literals, but require python >=3.6